src/components/InviteMemberProject/index.js

```diff
@@ -1,29 +1,60 @@
-import { Button, Container, Grid, IconButton, InputBase } from "@mui/material";
-import React, { useState } from "react";
-import { inviteMemberProjectSlice } from "../../state/inviteMemberProject/inviteMemberProjectSlice";
-import { Form } from "..";
 import SearchIcon from "@mui/icons-material/Search";
-import { NavLink } from "react-router-dom";
-import { useDispatch } from "react-redux";
+import {
+  Avatar,
+  Button,
+  Container,
+  Grid,
+  IconButton,
+  InputBase,
+  Stack,
+  Typography,
+} from "@mui/material";
+import { deepOrange } from "@mui/material/colors";
+import Paper from "@mui/material/Paper";
 import Table from "@mui/material/Table";
 import TableBody from "@mui/material/TableBody";
 import TableCell from "@mui/material/TableCell";
 import TableContainer from "@mui/material/TableContainer";
 import TableHead from "@mui/material/TableHead";
 import TableRow from "@mui/material/TableRow";
```

Comment:
Why different types of imports? Both named import and default import! Chose one. I suggest named one.

Reply:
updated